packages/financial-templates-lib/src/clients/InsuredBridgeL2Client.ts

```diff
@@ -122,12 +125,60 @@ export class InsuredBridgeL2Client {
     });
   }
 
-  async getFundsDepositedEvents(blockSearchConfig: { fromBlock: number; toBlock: number }): Promise<EventData[]> {
-    return await this.bridgeDepositBox.getPastEvents("FundsDeposited", blockSearchConfig);
+  async getFundsDepositedEvents(eventSearchOptions: EventSearchOptions): Promise<EventData[]> {
+    const eventsData = await getEventsForMultipleProviders(
+      [this.l2Web3].concat(this.fallbackL2Web3s),
+      this.bridgeDepositBox.options.jsonInterface,
```

Comment:
do you think it's better to call this using `getAbi("BridgeDepositBox")` or the way you have it now?

Reply:
i think using `getAbi` is more consistent thanks for highlighing